docs/sql-ref-syntax.md

```diff
@@ -19,4 +19,64 @@ license: |
   limitations under the License.
 ---
 
-Spark SQL is Apache Spark's module for working with structured data. The SQL Syntax section describes the SQL syntax in detail along with usage examples when applicable.
+Spark SQL is Apache Spark's module for working with structured data. The SQL Syntax section describes the SQL syntax in detail along with usage examples when applicable. This document provides a list of Data Definition and Data Manipulation Statements, as well as Data Retrieval and Auxiliary Statements.
+
+### Data Definition Statements
```

Comment:
How about `DDL Statements` for consistency? https://github.com/apache/spark/blob/master/docs/sql-migration-guide.md#ddl-statements